tasks.py

```diff
@@ -15,263 +15,269 @@
-    print(f"Successfully removed image {name}-{python_ver}")
+    print(f"Building image {name}:{image_ver}")
+    command = f"docker build --tag {name}:{image_ver} --build-arg PYTHON_VER={python_ver} -f Dockerfile ."
+
+    if nocache:
+        command += " --no-cache"
+    if forcerm:
+        command += " --force-rm"
+
+    result = context.run(command, hide=False)
+    if result.exited != 0:
+        print(f"Failed to build image {name}:{image_ver}\nError: {result.stderr}")
 
 
 @task
-def clean_images(context):
-    """This will remove the Python 3.6 and 3.7 images.
+def clean_image(context, name=NAME, image_ver=IMAGE_VER):
+    """This will remove the specific image.
 
     Args:
         context (obj): Used to run specific commands
+        name (str): Used to name the docker image
+        image_ver (str): Define image version
     """
-    _clean_image(context, NAME, "3.6")
-    _clean_image(context, NAME, "3.7")
+    print(f"Attempting to forcefully remove image {name}:{image_ver}")
+    context.run(f"docker rmi {name}:{image_ver} --force")
+    print(f"Successfully removed image {name}:{image_ver}")
 
 
 @task
-def rebuild_docker_images(context):
-    """This will clean the images for both Python 3.6 and 3.7 and then rebuild containers without using cache.
+def rebuild_image(context, name=NAME, python_ver=PYTHON_VER, image_ver=IMAGE_VER):
+    """This will clean the image and then rebuild image without using cache.
 
     Args:
         context (obj): Used to run specific commands
+        name (str): Used to name the docker image
+        python_ver (str): Define the Python version docker image to build from
+        image_ver (str): Define image version
     """
-    clean_images(context)
-    build_test_containers(context)
+    clean_image(context, name, image_ver)
+    build_image(context, name, python_ver, image_ver)
 
 
 @task
-def pytest(context, name=NAME, python_ver=PYTHON_VER):
+def pytest(context, name=NAME, image_ver=IMAGE_VER, local=INVOKE_LOCAL):
     """This will run pytest for the specified name and Python version.
 
     Args:
         context (obj): Used to run specific commands
         name (str): Used to name the docker image
-        python_ver (str): Will use the Python version docker image to build from
+        image_ver (str): Will use the container version docker image
+        local (bool): Define as `True` to execute locally
     """
     # pty is set to true to properly run the docker commands due to the invocation process of docker
     # https://docs.pyinvoke.org/en/latest/api/runners.html - Search for pty for more information
     # Install python module
-    docker = f"docker run -it -v {PWD}:/local {name}-{python_ver}:latest"
-    context.run(
-        f"{docker} /bin/bash -c 'poetry install && pytest "
-        "--cov=diffsync --cov-config pyproject.toml --cov-report html --cov-report term -vv'",
-        pty=True,
-    )
+    exec_cmd = "pytest -vv"
```

Comment:
Looks like you dropped the use of `coverage` here - intentional?

Reply:
good catch, I added it back, sorry about that